add-on/panels/js/roomViews.jsx

```diff
@@ -401,9 +401,15 @@ loop.roomViews = (function(mozL10n) {
       }
 
       // Automatically start sharing a tab now we're ready to share.
+      if (this.state.roomState === ROOM_STATES.MEDIA_WAIT &&
+          nextState.roomState === ROOM_STATES.JOINING) {
+        this.props.dispatcher.dispatch(new sharedActions.StartBrowserShare());
```

Comment:
Does this successfully share the tab? I believe dispatching this action eventually passes the window to the sdk, but if the sdk isn't set up per `SESSION_CONNECTED` state, I would expect tab sharing to fail.